documentation/docs/05-load.md

```diff
@@ -93,85 +146,176 @@ An instance of [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), co
+
+### Making fetch requests
 
-`fetch` is equivalent to the [native `fetch` web API](https://developer.mozilla.org/en-US/docs/Web/API/fetch), with a few additional features:
+Inside server-only `load` functions you can directly query your data base. Inside shared `load` functions you can't, as they also may run on the client. Also sometimes your data base may not be on the same server as your app. For these reasons it is often necessary to make `fetch` requests.
+
+The `load` function provides a `fetch` method which is equivalent to the [native `fetch` web API](https://developer.mozilla.org/en-US/docs/Web/API/fetch), with a few additional features:
 
 - it can be used to make credentialed requests on the server, as it inherits the `cookie` and `authorization` headers for the page request
 - it can make relative requests on the server (ordinarily, `fetch` requires a URL with an origin when used in a server context)
 - internal requests (e.g. for `+server.js` routes) go direct to the handler function when running on the server, without the overhead of an HTTP call
-- during server-side rendering, the response will be captured and inlined into the rendered HTML. Note that headers will _not_ be serialized, unless explicitly included via [`filterSerializedResponseHeaders`](/docs/hooks#server-hooks-handle)
-- during hydration, the response will be read from the HTML, guaranteeing consistency and preventing an additional network request
+- during server-side rendering, the response will be captured and inlined into the rendered HTML. Note that headers will _not_ be serialized, unless explicitly included via [`filterSerializedResponseHeaders`](/docs/hooks#server-hooks-handle). Then, during hydration, the response will be read from the HTML, guaranteeing consistency and preventing an additional network request - if you got a warning in your browser console when using the browser `fetch` instead of the `load` `fetch`, this is why.
 
 > Cookies will only be passed through if the target host is the same as the SvelteKit application or a more specific subdomain of it.
 
-#### parent
+### Headers and cookies
+
+A `load` function can set headers and cookies on the server.
 
-`await parent()` returns data from parent layout `load` functions. In `+page.server.js` or `+layout.server.js` it will return data from `load` functions in parent `+layout.server.js` files:
+If you need to set headers for the response, you can do so using the `setHeaders` method. This is useful if you want the page to be cached, for example:
+
+```js
+// @errors: 2322
+/// file: src/routes/blog/+page.server.js
+/** @type {import('./$types').PageServerLoad} */
+export async function load({ fetch, setHeaders }) {
+	const url = `https://cms.example.com/articles.json`;
+	const response = await fetch(url);
+
+	setHeaders({
+		age: response.headers.get('age'),
+		'cache-control': response.headers.get('cache-control')
+	});
```

Comment:
This doesn't actually work right now, see #7125 and #6477

Reply:
Yeah I took this snippet from the old docs and later realized that, too. Not sure how important it is to switch to another example given we want to support this eventually. cc @Rich-Harris